### PR TITLE
feat: Order steps are missing tax information.

### PR DESCRIPTION
**Pull request by iamdevvalecha**
Fixes https://github.com/fossasia/open-event-frontend/issues/7308

#### Short description of what this resolves:
tax info in details is added to order summary page.

#### Changes proposed in this pull request:

https://user-images.githubusercontent.com/71969867/119646517-e12bee80-be3c-11eb-84e2-e2abf15bf1f0.mp4

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by MeghalBisht** [comment]
Show the price as well and the same UI as ticket page

**Comment by iamdevvalecha** [comment]
previous:-

Screenshot from 2021-05-28 14-28-53

after changes:-
Screenshot from 2021-05-28 15-22-34

**Comment by iamdevvalecha** [comment]
@MeghalBisht check!

**Comment by iamdevvalecha** [comment]
@mariobehling @SundaramDubey @MeghalBisht please review!

File: app/templates/components/orders/order-summary.hbs [hunk 1]
```diff
@@ -110,7 +110,11 @@
                         {{t 'The price is inclusive of all taxes.'}}
                       {{else}}
                         {{#if ticket.price}}
-                          {{this.event.tax.rate}}%
+                        <td><CurrencyAmount @currency={{this.eventCurrency}} @amount={{sub ticket.ticketPriceWithTax ticket.price}}/>
+                 <small class="ui gray-text small">
+                          ({{this.event.tax.name}} {{this.event.tax.rate}}%)
+                        </small>
```
Comment (maze-runnar) [inline app/templates/components/orders/order-summary.hbs:116]: 6th time I guess - indentation.  [how to format code](https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode)

Comment (iamdevvalecha) [reply app/templates/components/orders/order-summary.hbs:116]: sorry it is do code formatter in my sublime.... and it is previous commit.

File: app/templates/components/orders/order-summary.hbs [hunk 2]
```diff
@@ -121,7 +125,10 @@
                     <Orders::TicketPrice
                       @ticket={{ticket}}
                       @currency={{this.eventCurrency}}
-                      @amount={{mult (sub ticket.price ticket.discount) tickets.length}} />      
+                      @amount={{mult (sub ticket.price ticket.discount) tickets.length}} />
+                         <small class="ui gray-text small">
+                  +<CurrencyAmount @currency={{this.eventCurrency}} @amount={{sub ticket.ticketPriceWithTax ticket.price}}/>
+                      </small>
```
Comment (maze-runnar) [inline app/templates/components/orders/order-summary.hbs:131]: fix indentation.

Comment (iamdevvalecha) [reply app/templates/components/orders/order-summary.hbs:131]: ok bhaiya.

File: app/templates/components/orders/order-summary.hbs [hunk 3]
```diff
@@ -110,7 +110,11 @@
                         {{t 'The price is inclusive of all taxes.'}}
                       {{else}}
                         {{#if ticket.price}}
-                          {{this.event.tax.rate}}%
+                    <td><CurrencyAmount @currency={{this.eventCurrency}} @amount={{sub ticket.ticketPriceWithTax ticket.price}}/>
+                        <small class="ui gray-text small">
+                          ({{this.event.tax.name}} {{this.event.tax.rate}}%)
+                        </small>
+                    </td>
```
Comment (maze-runnar) [inline app/templates/components/orders/order-summary.hbs:117]: move this block inside if, with proper `indentaton`

**Comment by iamdevvalecha** [comment]
@mariobehling review please.

**Comment by iamdevvalecha** [comment]
When tax is not included in ticket price:-
Screenshot from 2021-06-12 01-28-07
When tax is included in ticket price:-
Screenshot from 2021-06-12 00-46-07

**Comment by iamdevvalecha** [comment]
@MeghalBisht @maze-runnar @mariobehling please review!

**Comment by MeghalBisht** [comment]
@iamdevvalecha Despite repetitive reviews you still push code with wrong indentation. Now, with this comment this PR is getting reviewed 4th time for the indentation thing. This just increases the number of request review counts for a trivial issue and consumes a lot of time than it should deserve.

**Comment by MeghalBisht** [comment]
> When tax is included in ticket price:-
> Screenshot from 2021-06-12 00-46-07

When tax is included I don't think we should show it separately else what will be the difference in the two cases. I am not sure.

File: app/templates/components/orders/order-summary.hbs [hunk 4]
```diff
@@ -121,8 +126,22 @@
...
+                    {{#if this.event.tax.isTaxIncludedInPrice}}
+                      <small class="ui gray-text small">
+                        {{t 'includes'}} <CurrencyAmount @currency={{this.eventCurrency}} @amount={{ticket.includedTaxAmount}}/>
+                      </small>
+                    {{else}}
+                      <small class="ui gray-text small">
+                        + <CurrencyAmount @currency={{this.eventCurrency}} @amount={{sub ticket.ticketPriceWithTax ticket.price}}/>
+                      </small>
+                    {{/if}}
+                    <span>
```
Comment (MeghalBisht) [inline app/templates/components/orders/order-summary.hbs:140]: indentation

File: app/templates/components/orders/order-summary.hbs [hunk 5]
```diff
@@ -121,8 +126,22 @@
                     <Orders::TicketPrice
                       @ticket={{ticket}}
                       @currency={{this.eventCurrency}}
-                      @amount={{mult (sub ticket.price ticket.discount) tickets.length}} />      
-                  </td>
+                      @amount={{mult (sub ticket.price ticket.discount) tickets.length}} />
+                    <p>
+                    {{#if this.event.tax.isTaxIncludedInPrice}}
```
Comment (MeghalBisht) [inline app/templates/components/orders/order-summary.hbs:131]: indentation

**Comment by iamdevvalecha** [comment]
now?

**Comment by MeghalBisht** [comment]
Wrong

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
please increase width of tax column.
Don't show tax info (0.00) if tax in not enabled.

**Review by maze-runnar — APPROVED** [review]

**Review by MeghalBisht — APPROVED** [review]

**Pull request merged**